content/guidelines/reviewing.en.md

```diff
@@ -61,7 +61,12 @@ Your focus should be on two aspects: accuracy and quality.
 7. Verify the version of the document in the contribution tool by clicking on the _Verify version_ button.
 8. Ensure that all checks generated by the OTA-bot are manually checked.
 9. When you confirm that the term contribution has been made to the correct collection, proceed to add a review.
-10. Merge the contribution.
+
+## Merging the Pull Request
+
+Status checks are required to pass before merging can take place. This ensures that the Continuous Integration (CI) is completed successfully before merging the changes into the protected branch (main).
```

Comment:
> This might be a bit too much detail, maybe we can reduce the technicality for reviewers:
```suggestion
Status checks are required to pass before merging can take place. This ensures that automated tests ( through “Continuous Integration”, or CI) confirm the contribution will be readable by the Open Terms Archive engine.
```